src/tests/test_utils/common_test_utils/include/common_test_utils/test_enums.hpp

```diff
@@ -60,7 +60,8 @@ enum EltwiseTypes {
     BITWISE_AND,
     BITWISE_NOT,
     BITWISE_OR,
-    BITWISE_XOR
+    BITWISE_XOR,
+    EQUAL_OP
```

Comment:
Lets avoid extension of this enum given we alreasdy have ComparisonTypes

Reply:
checked: JIT emitter is used by `ComparisonTypes`. Additional test was removed.

Reply:
checked: JIT emitter is used by `ComparisonTypes`. Additional test was removed.